### ISSUE TITLE
The '/Give' Command

### ISSUE THREAD

**Issue by AnimeZer0** [Deathly/TerrariaAPI#11]
Hello There, I would like some help with the /give command.

As instructed i place the code in the onchat hook but it does nothing but spam the game with sounds and crash if you put in too much numbers.

Could the tutorial for it be more specific? Like what do i need to typ to give myself items?
/give -name- -item- -amount-
/give -item- -amount- 

or anything else?

Thanks
-AnimeZer0

**Comment by AlbinoDrought** [comment]
The parameters of the command would be "/give -item name- -amount-"
It can sometimes do weird things if the item name has spaces and you also specify an amount, but it's easy to live with.
That sound problem could be due to using the give command with a for loop in it, which I don't really like to use because there's a  built-in stack command. Try this code: http://pastebin.com/aQHgVZZb

**Comment by lioncash** [comment]
That code doesn't seem to work, nothing gets added to the inventory when used, unless I have it implemented wrong of course.

**Comment by lioncash** [comment]
Nope, nevermind, I was just a dolt. Forgot to put a capital letter for the beginning of the item name
